### PR TITLE
Feat: Added Sidebar

### PR DESCRIPTION
**Pull request by spraveenofficial**
Added Sidebar

### CODE REVIEW

File: frontend/src/Pages/Video/style.css [hunk 1]
```diff
@@ -57,6 +58,7 @@
 @media (max-width: 900px) {
   .video {
     padding: 20px;
+    margin-left: 0px;
```
Comment (mnath30) [inline frontend/src/Pages/Video/style.css]: I guess for 0 value you don't have to specify the unit

**Review by mnath30 — APPROVED** [review]
Code is clean and easy to read 
Looks good to me

File: frontend/src/Pages/Home/style.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  min-height: 100vh;
+}
+.homepage-items {
+  display: flex;
+  flex-direction: column;
+  scrollbar-width: thin;
+  width: 100%;
+  float: right;
+  margin-left: 250px;
+  /* min-height: 100vh; */
```
Comment (ArjunGTX) [inline frontend/src/Pages/Home/style.css:12]: Remove commented code

File: frontend/src/Components/Navbar/style.css [hunk 3]
```diff
@@ -6,6 +6,17 @@ nav {
   justify-content: space-between;
   align-items: center;
   text-align: center;
+  position: fixed;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  -webkit-transition: top 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
```
Comment (ArjunGTX) [inline frontend/src/Components/Navbar/style.css:13]: Great that you are using vendor prefixes for browser support

File: frontend/src/App.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ import Video from "./Pages/Video";
 import Loader from "./Components/Loader";
 import Upload from "./Pages/Upload";
 import Channel from "./Pages/Channel";
```
Comment (ArjunGTX) [inline frontend/src/App.js:12]: Suggestion: Can't these Components be imported from an index file from their respective folders?

**Review by ArjunGTX — APPROVED** [review]

File: frontend/src/App.js (hunk 4 above)
Comment (spraveenofficial) [reply frontend/src/App.js:12]: It can, will do all those in last.

File: frontend/src/Pages/Video/style.css (hunk 1 above)
Comment (spraveenofficial) [reply frontend/src/Pages/Video/style.css]: Thanks, removed.

**Pull request merged**